src/member/templates/client/view/payment.html

```diff
@@ -18,15 +18,32 @@ <h2 class="ui header">
             <div class="sub header">{% trans "Billing and payment information" %}.</div>
         </div>
     </h2>
+    <div class="ui right small floated menu">
+        <a class="edit icon item" href="{% url 'member:member_update_payment_information' client_id=client.id %}">
+            <i class="icon edit"></i>{% trans "Edit" %}
+        </a>
+    </div>
```

Comment:
Rather use this markup for all the Edit buttons:

```
<h2 class="ui header">
    <div class="content">
        {% trans 'Billing' %}
        <div class="sub header">{% trans "Billing and payment information" %}.</div>
    </div>
    <a class="ui right floated button" href="{% url 'member:member_update_payment_information' client_id=client.id %}">
        <i class="edit icon"></i>
        {% trans "Edit" %}
    </a>
</h2>
```

The _float_ is set on the anchor, wrapped in the <h2>